.github/workflows/build_and_push.yml

```
@@ -63,11 +118,9 @@ jobs:
 
       - name: Set CI Tag
         run: |
-          BRANCH="${GITHUB_REF##*/}"
-          CI_TAG=${BRANCH#v}-ci
-          if [ ${BRANCH} = "develop" ]; then
-            CI_TAG="ci"
-          fi
+          ROOT_DIR:=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+          CHART_YAML:=${ROOT_DIR}/deploy/helm/charts/Chart.yaml
+          CI_TAG := $(shell awk -F': ' '/^version:/ {print $$2}' $(CHART_YAML))
```

Just hardcode it here.. Not sure this works with the variables.
```suggestion
          CI_TAG := $(shell awk -F': ' '/^version:/ {print $$2}' deploy/helm/charts/Chart.yaml)
```

---

I just googled it and it might work. Let's still hardcode it.